lib/entities/main_actions.dart

```diff
@@ -48,7 +48,7 @@ class MainActions {
         case WalletType.bitcoin:
         case WalletType.litecoin:
           if (viewModel.isEnabledBuyAction) {
-            if (DeviceInfo.instance.isMobile) {
+            if (ResponsiveLayoutUtil.instance.isMobile) {
```

Comment:
revert to Device Info, as the in-app webview package is not supported except by iOS and Android